paper/report_final.tex

```diff
@@ -0,0 +1,563 @@
+average brain activation in all three runs. For each voxel $i$, we fit the following mixed-effects models, note that here we only include the intercept term for random effects. 
+
+\begin{equation}
+Y_{i, k} = \beta_{i, 0} + \beta_{i,1} *X_{ldrift} + \beta_{i, 2} * X_{qdrift} +  \beta_{i, loss} *X_{loss} + \beta_{i, gain} * X_{gain}  + \gamma _{i, k} + \epsilon_{i, k}, \quad k =1, 2, 3
+\end{equation}
+
+Then we calculate the neural loss aversion level and plot heat maps in the same way as the about section for multiple linear regressions and compare the results of two models. 
+
+\subsubsection{Whole brain analysis of correlation between 
+neural activity and behavioral response across participants}
+
+We then apply the above model on the standard brain to analysis the neural 
+activity and behavioral response across participants. For each participant, 
+we pick up several regions with highest activation level, calculate the mean 
+neural loss aversion $\bar{\eta}$ within these specific region. Thus we could 
+examine the relationship between neural activity and behavioral using the 
+following regression model:
+
+\begin{equation}
+\lambda = \alpha_0 + \alpha_1 * \eta + \epsilon
+\end{equation}
+
+where the sample size is the number of participants(16).
+
+\subsection{Methods}
+
+\subsubsection{Cross-validation}
+
+We fit linear models for each voxel for each participant. For each linear
+model, we do a k-fold cross-validation. Since the sample size for each linear 
+regression model range from 80-90, we choose to use 10 fold cross-validation,
+which means the original sample is randomly pertitioned into 10 equal sized 
+subsamples. \\
+In the behavioral analysis using Logistic regression, since the responce 
+variables are binary, we calculate the misclassification error rate to 
+summarize the fit. In the neural linear regression model using BOLD data, we 
+use the mean squared error to summarize the errors.
+
+\subsubsection{ROC curve}
+
+\subsubsection{Inferences on data}
+
+After fitting regression models on our BOLD and behavioral data, we would try 
+assessing and validating our models. In order to do this, we would calculate 
+for the residual sum of squares for our model. We have to do three tests for 
+the model. The first one is that we calculate the t-statistics and p-value for 
+our beta coefficients to check whether our beta parameters are statistically 
+significant at a significance level of 5\%. The second one is that we calculate
+the residuals of this linear model and check whether it follows a normal 
+distribution. The third one is that we calculate the R-Squared value and the 
+adjusted R-squared value to see whether the values are good for the linear 
+regression model.\\
+
+\begin{itemize}
+\item{QQ plot}
+\item{Residual - fitted plot}
+\end{itemize}
+
+
+\subsubsection{Multiple Test correction}
+
```

Comment:
we're adding to the ROC, multiple test, cluster analysis, and anova subsubsections right?